openvdb_houdini/openvdb_houdini/SOP_OpenVDB_AX.cc

```diff
@@ -853,35 +870,7 @@ void SOP_OpenVDB_AX::syncNodeVersion(const char* old_version,
             }
         }},
         {
-        "0.3.0", {
-            { "attributestocreate",
-                [](const SOP_OpenVDB_AX& node) -> std::string {
-                    const bool createMissing = static_cast<bool>(node.evalInt("createmissing", 0, 0));
-                    if (createMissing == 1) return "*";
-                    else return "";
-                }
-            }
-        }},
-        {
-        "1.0.0", {
-            { "attributestocreate",
-                [](const SOP_OpenVDB_AX& node) -> std::string {
-                    const bool createMissing = static_cast<bool>(node.evalInt("createattributes", 0, 0));
-                    if (createMissing == 1) return "*";
-                    else return "";
-                }
-            },
-        }},
-        {
-        "8.0.0", {
-            // ax, ax sop and vdb versions re-synced at this version
-            { "attributestocreate",
-                [](const SOP_OpenVDB_AX& node) -> std::string {
-                    const bool createMissing = static_cast<bool>(node.evalInt("createattributes", 0, 0));
-                    if (createMissing == 1) return "*";
-                    else return "";
-                }
-            },
+        "8.0.2", {
```

Comment:
8.0.2 is now 8.1.0

Reply:
Still needs to be 8.0.2 as this is when this change was introduced (i.e. `OPENVDB_LIBRARY_PATCH_VERSION_NUMBER` was 2 at this point in time). This system will apply this change to anything syncing before this version to this version (inclusive) and later

Reply:
Ok, that makes sense.